src/js/classes/Map.js

```diff
@@ -175,7 +176,7 @@ class Map {
 
     this.projection = d3.geoMercator().scale(5500).center([-75.75, 40.5]);
 
-    this.renderTooltip = configureTooltip({...tooltipSchema, trigger: "manual", placement: "top"})
+    this.renderTooltip = configureTooltip({...tooltipSchema, trigger: "manual", placement: "top"});
```

Comment:
Wondering why its set to `trigger: "manual"`. I removed this in my branch but happy to change if this is here for a reason.

Reply:
Needed this for the side-by-side race maps. When you hover one county, the tooltip should show for both that county and the county in other map - I remember having trouble relying on default behavior.

If you removed that and everything works, then it's probably fine to remove.

Reply:
ah gotcha, well it's working for me without it

Reply:
Cool, let's take your change in your PR removing it and add it back if needeed